### DISCUSSION TITLE
Ability to edit Scheduled "Approved" revisions

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Revisions#964] [Feature Requests]
It will not be possible to edit scheduled "Approved" revisions. It will be helpful if we support it

Usecase:

> I am using PublishPress to schedule updates for my posts. Sometimes I notice mistakes after approval, so I would be very happy if this feature is added.

https://wordpress.org/support/topic/i-would-like-to-edit-an-article-that-has-already-been-approved/#post-16659998

**Comment by KratosGemini** [comment]
I've edited approved revisions before, so this may have been implemented. Or does it just work because I'm an Administrator on the site?
